test/README.md

```diff
@@ -23,3 +23,103 @@ Some examples of test harnesses using these tests, from the source repository:
 A [JSON schema](./schemas/) is included for the test files in this repository.
 
 For users of Visual Studio Code, a [settings file](./.vscode/settings.json) is included that enables schema validation while editing the test files.
+
+## Test Functions
+
+As the behaviour of some of the default registry _functions_
+such as `:number` and `:datetime`
+is dependent on locale-specific data and may vary between implementations,
+the following _functions_ are defined for **test use only**:
+
+### `:test:function`
+
+This function is valid both as a _selector_ and as a _formatter_.
+
+#### Operands
+
+The function `:test:function` requires a [Number Operand](/spec/registry.md#number-operands) as its _operand_.
+
+#### Options
+
+The only _option_ `:test:function` recognizes is `fd`,
+a _digit size option_ for which only `0` and `1` are valid values.
```

Comment:
Linkifying digit size just as "Number Operand" is being linkified 5 lines above. Helps short circuit questions like mine for the reader.

```suggestion
a [_digit size option_](/spec/registry.md#digit-size-options) for which only `0` and `1` are valid values.
```

Reply:
Underscore italics *are* linkified when the spec is turned into HTML. That's what [this](https://github.com/unicode-org/message-format-wg/blob/main/docs/tools/linkify.js) is for.